test/blackbox-tests/test-cases/github4814.t/run.t

```diff
@@ -1,9 +1,10 @@
 Issue #4814:
 When multiple packages are present, `dune build -p PKG` and `dune install -p
 PKG` should be able to build and install only that package, instead of all.
+
   $ dune build -p pkg1
-  $ dune install -p pkg1
-  Error: The following <package>.install are missing:
-  - _build/default/pkg2.install
-  Hint: try running 'dune build [-p <pkg>] @install'
-  [1]
+
+This is now fixed, although we should really just not allow -p on $ dune
```

Comment:
Now that `Dune_load.packages` returns the right packages, this issue has been fixed "by accident".